### PR TITLE
Add support for application tags and install params.

### PR DESCRIPTION
**Pull request by nerdguyahmad**
## Summary

This pull request adds support for `tags`, `custom_install_url` and `install_params` on `AppInfo`.

## Checklist

- [x] If code changes were made then they have been tested.
    - [x] I have updated the documentation to reflect the changes.
- [ ] This PR fixes an issue.
- [x] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

File: discord/appinfo.py [hunk 1]
```diff
@@ -107,6 +110,18 @@ class AppInfo:
     privacy_policy_url: Optional[:class:`str`]
         The application's privacy policy URL, if set.
 
+        .. versionadded:: 2.0
```
Comment (diceroll123) [inline discord/appinfo.py:113]: Need new lines in between versionadded + the following attribute

File: discord/appinfo.py [hunk 2]
```diff
@@ -266,3 +289,27 @@ def flags(self) -> ApplicationFlags:
...
+        The list of `OAuth2 scopes <https://discord.com/developers/docs/topics/oauth2#shared-resources-oauth2-scopes>`_
+        to add the application to a guild with.
+    permissions: :class:`Permissions`
+        The permissions to give to application in the guild.
+    """
+
+    __slots__ = (
+        "scopes",
+        "permissions",
+    )
```
Comment (Rapptz) [inline discord/appinfo.py:311]:
```suggestion
    __slots__ = ('scopes', 'permissions')
```

File: discord/appinfo.py [hunk 3]
```diff
@@ -266,3 +289,27 @@ def flags(self) -> ApplicationFlags:
...
+
+    def __init__(self, data: InstallParamsPayload) -> None:
+        self.scopes: List[str] = data.get("scopes", [])
```
Comment (Rapptz) [inline discord/appinfo.py:314]:
```suggestion
        self.scopes: List[str] = data.get('scopes', [])
```

File: discord/appinfo.py [hunk 4]
```diff
@@ -266,3 +289,27 @@ def flags(self) -> ApplicationFlags:
...
+        self.permissions: Permissions = Permissions._from_value(int(data["permissions"]))
```
Comment (Rapptz) [inline discord/appinfo.py:315]:
```suggestion
        self.permissions: Permissions = Permissions(int(data['permissions']))
```

File: discord/appinfo.py [hunk 5]
```diff
@@ -157,6 +175,11 @@ def __init__(self, state: ConnectionState, data: AppInfoPayload):
         self._cover_image: Optional[str] = data.get('cover_image')
         self.terms_of_service_url: Optional[str] = data.get('terms_of_service_url')
         self.privacy_policy_url: Optional[str] = data.get('privacy_policy_url')
+        self.tags: List[str] = data.get("tags", [])
```
Comment (Rapptz) [inline discord/appinfo.py:178]:
```suggestion
        self.tags: List[str] = data.get('tags', [])
```

File: discord/appinfo.py [hunk 6]
```diff
@@ -157,6 +175,11 @@ def __init__(self, state: ConnectionState, data: AppInfoPayload):
...
+        self.custom_install_url: Optional[str] = data.get("custom_install_url")
```
Comment (Rapptz) [inline discord/appinfo.py:179]:
```suggestion
        self.custom_install_url: Optional[str] = data.get('custom_install_url')
```

File: discord/appinfo.py [hunk 7]
```diff
@@ -157,6 +175,11 @@ def __init__(self, state: ConnectionState, data: AppInfoPayload):
...
+
+        params = data.get("install_params")
```
Comment (Rapptz) [inline discord/appinfo.py:181]:
```suggestion
        params = data.get('install_params')
```

**Review by Rapptz — CHANGES_REQUESTED** [review]
Looks good, mostly just stylistic changes.

File: discord/appinfo.py (hunk 1 above)
Comment (Rapptz) [reply discord/appinfo.py:113]: Not actually necessary

Comment (nerdguyahmad) [reply discord/appinfo.py:113]: Just to remain consistent with doc of other attributes, I've added them

**Review by Rapptz — APPROVED** [review]
LGTM.

**Pull request merged**